gtsfm/runner/run_vis_tool.py

```diff
@@ -0,0 +1,22 @@
+"""Runner for the visualization tool only (back-end results are not desired or required).
```

Comment:
this is technically not a runner since it does not inherit from the runner base class. It might be good to  to reserve "runner" to anything that runs SfM. Could we call this file something else? And maybe move it to the viz folder?